src/utils/v2v3/fundingCycle.ts

```diff
@@ -138,19 +139,21 @@ export const deriveNextIssuanceRate = ({
   weight,
   previousFC,
```

Comment:
non blocking but wonder if its just easier to pass in exactly what we need, rather than the whole fc object 🤔

Reply:
Good call, should definitely have just made this function take `oldWeight` and `oldDiscountRate` as opposed to the whole FC object. However, changing it will create quite a diff - perhaps not appropriate for this PR. I will get to it in a follow up

<img width="409" alt="Screen Shot 2023-09-09 at 3 06 29 pm" src="https://github.com/jbx-protocol/juice-interface/assets/96150256/4cf857ee-3d0a-4086-87bb-28574a0850d8">